protos/protos.go

```diff
@@ -126,7 +126,7 @@ func (p Protocol) String() string {
 }
 
 type Protocols interface {
-	BpfFilter(with_vlans bool) string
+	BpfFilter(with_vlans bool, with_icmp bool) string
```

Comment:
I would suggest to name it withIcmp. I'm aware that there are still quite a few variables that use _ but we should migrate to the new naming standard camelCase for the new variables (in case it doesn't break something).